### PR TITLE
Add nightly support and PHP 7.4

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add 7.4 support for nightly builds and don't erase directories if already exist
| Type?         | improvement

### CODE REVIEW

File: prestashop_docker/tag_manager.py [hunk 1]
```diff
@@ -35,14 +35,11 @@ def build(self, version=None):
                 'Building {}'.format(version)
             )
 
-            if self.exists(version):
```
Comment (PierreRambaud) [inline prestashop_docker/tag_manager.py:38]: Useless check

File: prestashop_docker/tag_manager.py [hunk 2]
```diff
@@ -35,14 +35,11 @@ def build(self, version=None):
...
-                continue
-
             log = self.docker_client.api.build(
                 path=str(version_path),
                 tag='prestashop/prestashop:' + version,
                 rm=True,
-                nocache=False,
+                nocache=True,
```
Comment (PierreRambaud) [inline prestashop_docker/tag_manager.py:42]: Don't use cache when building images

File: base/generate_tags.sh [hunk 3]
```diff
@@ -21,6 +21,10 @@ generate_image()
     fi
 
 
+    if [ -d images/$folder ]; then
+        return
```
Comment (PierreRambaud) [inline base/generate_tags.sh:25]:
```suggestion
        # Do not erase what we already defined in the directory
        return
```

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PierreRambaud

**Pull request merged**